### PR TITLE
Added kubectx plugin

### PR DESCRIPTION
**Pull request by hickey**
Entry for the kubectx plugin. Same as the others... passing CI testing and installs the `kubectx` and `kubens` scripts correctly.

### CODE REVIEW

**Comment by hickey** [comment]
I realized a little bit ago that there is already a MR for a kubectx plugin. I have looked at the repository pointed to for that MR and the only significant difference is that I setup the `kubens` script in the plugin which is part of the source package and used by many people.

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ itself) to look for plugins.
...
 | Kubectl                       | [Banno/asdf-kubectl](https://github.com/Banno/asdf-kubectl)                                       | [![Build Status](https://travis-ci.org/Banno/asdf-kubectl.svg?branch=master)](https://travis-ci.org/Banno/asdf-kubectl)                                                                |
 | kubectl-bindrole              | [looztra/asdf-kubectl-bindrole](https://github.com/looztra/asdf-kubectl-bindrole)                 | [![Build Status](https://travis-ci.org/looztra/asdf-kubectl-bindrole.svg?branch=master)](https://travis-ci.org/looztra/asdf-kubectl-bindrole)                                          |
+| kubectx                       | [wt0f/asdf-kubectx](https://gitlab.com/wt0f/asdf-kubectx.git)                                     | [![pipeline status](https://gitlab.com/wt0f/asdf-kubectx/badges/master/pipeline.svg)](https://gitlab.com/wt0f/asdf-kubectx/-/commits/master)                                           |
```
Comment (imbsky) [inline README.md]: The `.git` is not required here.
`[wt0f/asdf-kubectx](https://gitlab.com/wt0f/asdf-kubectx)`

**Comment by johnlayton** [comment]
g'day, i have updated the other kubectx plugin, however i noticed this, so i will remove from my pr.

**Comment by imbsky** [comment]
Thanks @hickey for your contribution!

**Pull request merged**